src/x/generated-source-files.mk

```diff
@@ -176,6 +176,38 @@ genny-leakcheckpool:
 	$(eval out_dir=$(gopath_prefix)/$(target_package))
 	cat $(m3x_package_path)/generics/leakcheckpool/pool.go | grep -v nolint | genny -pkg $(pkg) -ast gen "elemType=$(elem_type) elemTypePool=$(elem_type_pool)" > "$(out_dir)/$(out_file)"
 
+.PHONY: list-gen
+list-gen:
```

Comment:
nit: mind sticking to the `genny-` convention

maybe something like `genny-pooled-elem-list`?